vscode/src/commands/create/seahorse.ts

```diff
@@ -1,48 +1,48 @@
 import * as path from "path";
 
 import { PATHS } from "../../constants";
+import { Files, PgFs } from "../../utils";
 import { processCreateAnchor } from "./anchor";
-import { processCreate } from "./common";
 
 export const processCreateSeahorse = async () => {
   // Create Anchor
   const { name } = await processCreateAnchor();
 
-  // Create Seahorse
-  await processCreate({
-    name,
-    files: [
-      // Program <name>.py
-      [
-        path.join(PATHS.DIRS.PROGRAMS_PY, `${name}.py`),
-        `# ${name}
+  const seahorseFiles: Files = [
+    // Program <name>.py
+    [
+      path.join(PATHS.DIRS.PROGRAMS_PY, `${name}.py`),
+      `# ${name}
 # Built with Seahorse v0.2.3
 
 from seahorse.prelude import *
 
 declare_id('Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS')
 `,
-      ],
-
-      // __init__.py
-      [
-        path.join(
-          PATHS.DIRS.PROGRAMS_PY,
-          PATHS.DIRS.SEAHORSE,
-          PATHS.FILES.INIT_PY
-        ),
-        "",
-      ],
-
-      // prelude.py
-      [
-        path.join(
-          PATHS.DIRS.PROGRAMS_PY,
-          PATHS.DIRS.SEAHORSE,
-          PATHS.FILES.PRELUDE_PY
-        ),
-        require("./seahorse-prelude.py"),
-      ],
     ],
-  });
+
+    // __init__.py
+    [
+      path.join(
+        PATHS.DIRS.PROGRAMS_PY,
+        PATHS.DIRS.SEAHORSE,
+        PATHS.FILES.INIT_PY
+      ),
+      "",
+    ],
+
+    // prelude.py
+    [
+      path.join(
+        PATHS.DIRS.PROGRAMS_PY,
+        PATHS.DIRS.SEAHORSE,
+        PATHS.FILES.PRELUDE_PY
+      ),
+      require("./seahorse-prelude.py"),
+    ],
+  ];
+
+  const baseUri = await PgFs.getBaseUri();
```

Comment:
We should use the same `baseUri` that `processCreateAnchor` uses. Anchor could create a nested folder so there is no guarantee `baseUri` we get here is the same as the Anchor project uri.

Reply:
Thanks! Updated to return a `projectUri` from `processCreateAnchor` which is the URI seahorse should use.